common/app/model/PressedStory.scala

```diff
@@ -26,6 +28,16 @@ object PressedStory {
     val trail = Trail.make(tags, fields, commercial, elements, metadata, apiContent)
     val atoms = Atoms.make(apiContent)
     val sectionId: Option[SectionId] = metadata.section.map(s => SectionId(s.value))
+    val mediaAtoms = atoms.fold(Seq.empty[MediaAtom])(_.media)
+
+    val mainMediaAtom = {
+      val atomId =
+        Jsoup.parseBodyFragment(fields.main).getElementsByTag("gu-atom").asScala.headOption.map(_.attr("data-atom-id"))
```

Comment:
Do you think we should completely copy the logic [`facia` currently uses](https://github.com/guardian/frontend/blob/main/common/app/implicits/FaciaContentFrontendHelpers.scala#L31-L33) to find the `atomId`? It looks like we're missing a few steps as we traverse the html. Worth saying though I have no idea if / why these steps are needed. This bit of logic was introduced in https://github.com/guardian/frontend/pull/15513 and there is no justification.